app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -360,6 +360,34 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
     border: 1px solid var(--base-10);
   }
 
+  .current-user-info {
+    text-align: center;
+    margin-bottom: 1rem;
+
+    .current-user-avatar-container {
+      height: 80px;
+    }
+
+    .current-user-avatar {
+      border-radius: 100%;
+      height: inherit;
+      object-fit: cover;
+      border: 2px solid var(--base-90);
+    }
+
+    h3 {
+      margin-top: 6px;
+    }
+
+    p {
+      font-size: 16px;
+      height: 24px;
+      color: var(--base-70);
+      margin-top: -10px
+    }
+  }
+
+
```

Comment:
I also wonder if we can extract a lot of these styles into CSS variables here. Some of them feel like "magic numbers" to me